### DISCUSSION TITLE
Octopus Energy failed to set up after core update

### DISCUSSION THREAD

**Discussion by yebaws** [BottlecapDave/HomeAssistant-OctopusEnergy#607] [General]
Did core update last night and this morning I get "Failed to set up" for Octopus energy and all of my sensors. Any ideas on the best way to trouble shoot this? Not too keen to re-install if I don't have to, as I'll have a lot of target rate sensors and automations to set up again. Could always roll back the update from the last backup, but would rather resolve it now so it's not waiting to happen when I do update core in the future

**Comment by yebaws** [comment]
Re-downloaded it through HACS then re-started HA. Seems to have fixed it!

**Reply by BottlecapDave** [reply]
This was a failure on OE as their APIs were down (as shown in raised errors)

**Reply by willenic** [reply]
Just updated HA supervisor and lost all my entities too. Will try re-downloading....

Edited: all the entities back after re-downloading
